### PR TITLE
Add input var

### PR DESCRIPTION
**Pull request by viane**
Add input variable filed in the playbook input file for CPM provision.
Add default timeout to all API requests due to the customer may using a slow network z/OSMF.
Add zmf_body input parameter for using playbook in full automation.

### CODE REVIEW

**Review by shreeji818 — APPROVED** [review]

**Comment by viane** [comment]
@zosmf-Robyn Hi Robyn, could you check if the code satisfies your release criteria and give us clearance to merge if everything looks okay. Thanks

**Comment by zosmf-Robyn** [comment]
@viane Allen, please fix below issues in sanity test before merge:

$ ansible-lint roles/zmf_cpm_provision_software_service
[WARNING]: Unable to convert data using to_nice_json, falling back to to_json:
'full_instance_json' is undefined
[204] Lines should be no longer than 160 chars
roles/zmf_cpm_provision_software_service/tasks/main.yml:21
    zmf_body: '{"domain-name":"{{ domain_name }}","tenant-name":"{{ tenant_name }}","systems-nicknames":["{{ systems_nicknames }}"],"input-variables":{{ input_vars }}}'

**Comment by shreeji818** [comment]
@viane Allen, I added one more file (README_zmf_cpm_provision_software_service.rst) to your branch. Will you please review it and respond to Robyn's suggestions. Thanks.

**Review by shreeji818 — APPROVED** [review]

**Review by shreeji818 — APPROVED** [review]

**Review by shreeji818 — APPROVED** [review]

**Review by shreeji818 — APPROVED** [review]

**Review by shreeji818 — APPROVED** [review]

**Comment by viane** [comment]
@zosmf-Robyn @shreeji818 Just a reminder that this PR is verified and can be merged to upstream in the next release, also this PR includes the change request in PR #88 as well so feel free to close that one when done merging this PR. Thanks

File: docs/source/roles/README_zmf_cpm_provision_software_service.rst [hunk 1]
```diff
@@ -76,6 +76,16 @@ systems_nicknames
 
   | **required**: False
   | **type**: str
+  
+input_vars
```
Comment (viane) [inline docs/source/roles/README_zmf_cpm_provision_software_service.rst:80]: @shreeji818 already in here

**Pull request merged**